### PR TITLE
js and graphql snippets for create-wasm-wrappers sequence

### PR DESCRIPTION
**Pull request by krisbitney**
Added JavaScript and GraphQL snippets for pages in create-wasm-wrappers section.  Also added `.graphql` extension to the list of acceptable file types for snippet extraction.

### CODE REVIEW

File: snippets/as/simple-storage/run.sh [hunk 1]
```diff
@@ -0,0 +1,2 @@
+yarn --no-lockfile
+yarn build
```
Comment (dOrgJelli) [inline snippets/as/simple-storage/run.sh:2]: For CI purposes, we should make sure that the tests run as well

File: snippets/as/simple-storage/src/__tests__/e2e/types.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+// $start: js-e2e-test-types
+export type UInt8 = number;
```
Comment (dOrgJelli) [inline snippets/as/simple-storage/src/__tests__/e2e/types.ts:2]: We should probably use the `w3 app codegen` command to generate these types. Checkout an example here: https://github.com/polywrap/monorepo/tree/prealpha/packages/templates/api/assemblyscript/src/__tests__/types

**Review by dOrgJelli — CHANGES_REQUESTED** [review]

File: snippets/as/simple-storage/run.sh (hunk 1 above)
Comment (krisbitney) [reply snippets/as/simple-storage/run.sh:3]: They now run

File: snippets/as/simple-storage/src/__tests__/e2e/types.ts (hunk 2 above)
Comment (krisbitney) [reply snippets/as/simple-storage/src/__tests__/e2e/types.ts]: done

**Review by dOrgJelli — APPROVED** [review]

**Pull request merged**